src/main/java/dk/kb/present/DSCollection.java

```diff
@@ -164,6 +175,38 @@ public Stream<DsRecordDto> getDSRecords(Long mTime, Long maxRecords, String form
         }
     }
 
+    /**
+     * Extract children records from storage if present in input record.
+     * @param record     to extract children for.
+     * @return the string value of the related record for the input record.
```

Comment:
Please state that only the first record is handled and that the URI to the raw record is returned.